crypto/fipsmodule/sha/internal.h

```diff
@@ -61,6 +63,11 @@ extern "C" {
 #define SHA3_BLOCKSIZE(bitlen) (KECCAK1600_WIDTH - bitlen * 2) / 8
 #define SHA3_PAD_CHAR 0x06
 
+#define SHA3_224_CBLOCK SHA3_BLOCKSIZE(SHA3_224_DIGEST_BITLENGTH)
+#define SHA3_256_CBLOCK SHA3_BLOCKSIZE(SHA3_256_DIGEST_BITLENGTH)
+#define SHA3_384_CBLOCK SHA3_BLOCKSIZE(SHA3_384_DIGEST_BITLENGTH)
+#define SHA3_512_CBLOCK SHA3_BLOCKSIZE(SHA3_512_DIGEST_BITLENGTH)
```

Comment:
Nit: Should we move this with each `SHA3_*_CAPACITY_BYTES`, `SHA3_*_DIGEST_BITLENGTH`, etc. for consistency?

Reply:
sure, makes sense.